### ISSUE TITLE
Windows 10 support

### ISSUE THREAD

**Issue by flolilo** [piedar/espresso-python#2]
Hello!

A few infos:
 - **OS**:  Win 10 (v1803)
    - this corresponds to:
```python
sys.getwindowsversion(major=10, minor=0, build=17134, platform=2, service_pack='')
```
 - **Python**:  v3.7.4
 - **espresso-python**: latest (`045f856`)

----

As it is, using `python .\espresso.py` throws:

```
Traceback (most recent call last):
  File ".\espresso.py", line 81, in <module>
    icon = TrayIcon(inhibitors.AutoSelect(), app);
  File "D:\Downloads\espresso-python-master\inhibitors.py", line 21, in AutoSelect
    raise NotImplementedError("Platform not supported")
NotImplementedError: Platform not supported
```

So I changed `inhibitors.py`'s line `18` from:
```python
elif platform.system() == "Windows" and sys.getwindowsversion().major >= 6 and sys.getwindowsversion().minor >= 1:
```
to:
```python
elif platform.system() == "Windows" and (sys.getwindowsversion().major >= 6 and sys.getwindowsversion().minor >= 1) or sys.getwindowsversion().major >= 7:
```

Another possible solution would be to change it to `sys.getwindowsversion().build >= 7601`, where `7601` is Windows 7 SP 1.

----

It now seems to work, but I am still testing.

**Comment by flolilo** [comment]
Update: Found no problems so far, so I created a pull request: #3

**Comment by piedar** [comment]
Fixed by #4.
